check/controls.go

```diff
@@ -199,5 +227,24 @@ func summarizeGroup(group *Group, check *Check) {
 		group.Warn++
 	case INFO:
 		group.Info++
+	case SKIP:
+		group.Skip++
 	}
 }
+
+func summarizeLevel(control *Controls, check *Check) {
```

Comment:
I'm sure there must be a way to avoid having all these different incredibly similar summarize functions!